README.md

```diff
@@ -1,167 +1,169 @@
-**<!-- ALWAYS CLICK APPLY AFTER ENABLING OR DISABLING MODS! --!>**
-<br />
-![image](https://user-images.githubusercontent.com/23240514/152447480-e40d2b78-c020-4c58-8da2-f29b85a95e7e.png)<br /><br /><br /><br />
+## Installing Mods
+
+* You can see a list of all installed and disabled mods in the ``Mods`` section:
+![image](https://user-images.githubusercontent.com/23240514/152446534-9109f141-c98c-4046-b023-7451b4796d3e.png)
+
+* If you have a .zip file of the mod you want to install, you simply need to browse to the .zip file and   select it. It'll be installed in the same way.
 
-<h1> Installing Skins</h1>
+* If you want to enable, or disable mods, simply click the specific mod's name in the list, and press the corresponding arrow button to move them between the Enabled/Disabled lists.
 
+![image](https://user-images.githubusercontent.com/23240514/152447411-2f7b1f99-a558-4885-bd11-f45952d6a04b.png)
+![image](https://user-images.githubusercontent.com/23240514/152447434-f2996352-9237-46f0-aa07-2c2e3181ea21.png)
+![image](https://user-images.githubusercontent.com/23240514/152447448-24d51ad1-da48-44d2-b1d6-c864767cd230.png)
+* **ALWAYS CLICK ``APPLY`` AFTER ENABLING OR DISABLING MODS!**
+
+## Installing Skins 
+
+* First, Navigate to your desired Skin's ``.ZIP`` file
 ![image](https://user-images.githubusercontent.com/23240514/152447515-c4eb762b-d4b9-49da-b713-87aded2220d2.png)
 <br /><br />
 
-Click import skin and wait for the compatibliity check to finish.
-<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447613-e3fc58d8-992e-46ad-9c2c-499a87888603.png)<br /><br />
-![image](https://user-images.githubusercontent.com/23240514/152447631-03075684-204e-4311-b3dc-4bd6e264d0f3.png)<br /><br />
-Once the compatibility is confirmed, the application can now inject the skin.
-<br /><br />
-Click "Install Skin" to proceed with the installation of the skin.
-<br /><br />
+* Click import skin and wait for the compatibliity check to finish.
+
+![image](https://user-images.githubusercontent.com/23240514/152447613-e3fc58d8-992e-46ad-9c2c-499a87888603.png)
+
+![image](https://user-images.githubusercontent.com/23240514/152447631-03075684-204e-4311-b3dc-4bd6e264d0f3.png)
+
+* Once the compatibility is ``Confirmed`` with a green light, the application can now inject the skin.
+* Click ``Install Skin`` to proceed with the installation of the skin.
+* Installed Skins usually replace the ``Factory Issue`` skin of a weapon
+
 ![image](https://user-images.githubusercontent.com/23240514/152447644-27b7089a-1c17-4628-8616-42f8c0950baa.png)
-<br /><br />
-Skins will be previewed in the following way
-<br /><br />
+
+Skin Textures will be previewed in the following way
+
 ![image](https://user-images.githubusercontent.com/23240514/152447664-09a21668-b54f-41df-8493-f1567c497ec1.png)
-<br /><br /><br /><br />
 
-<h1>Updating</h1>
 
-<br /><br />
+## Updating VTOL
+
+* You cam check for updates in the ``Updates`` Section
```

Comment:
typo

Reply:
well cock, i missed this one, thanks for telling me